news/cookie.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Repo structure modified to the new diffpy standard
```

Comment:
add comment about code linting?

Are we changing the Python versions that are supported? T his may be the biggest change.

Reply:
Added